_data/sites.json

```
@@ -135,6 +135,16 @@
         ]
     },
 
+    {
+        "name": "Adclerks",
+        "url": "https://adclerks.com/privacy",
```

```suggestion
        "url": "https://adclerks.com/privacy",
        "email": "support@adclerks.com",
        "email_subject": "Personal Data Request",
```
Since you have detailed instructions on how to request it by e-mail, might as well fill-in the e-mail and e-mail subject fields.